scripts/benchmark.bat

```diff
@@ -44,5 +44,12 @@ for /l %%i in (0, 1, 0) do (
 
 echo.
 echo Benchmarking...
-zig\zig.exe build benchmark -Drelease-safe
-exit /b %errorlevel%
+
+set ARGS=
+for %%a in (%*) do (
```

Comment:
I'm so glad I didn't try to figure this out myself :D

Reply:
the tricky part was the label's name `:main` included in the argument list `%*` ... it makes NO sense at all!!!